test/e2e-go/cli/algoh/expect/algoh_expect_test.go

```diff
@@ -19,10 +19,12 @@ import (
 	"testing"
 
 	"github.com/algorand/go-algorand/test/framework/fixtures"
+	"github.com/algorand/go-algorand/test/partitiontest"
 )
 
 // TestAlgohWithExpect Process all expect script files with suffix Test.exp within the test/e2e-go/cli/algoh/expect directory
 func TestAlgohWithExpect(t *testing.T) {
+	partitiontest.PartitionTest(t)
```

Comment:
This should be be called here because it is already partitioned in `et.Run()` with better partitioning. 
Partitioning here will result in double partitioning, leaving many tests out.